spec/linkedList.js

```diff
@@ -0,0 +1,213 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import LinkedList from '../src/linkedList'
+
+chai.use(chaiChange)
+
+describe.only('LinkedList', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(LinkedList).to.be.a('function')
+  })
+
+  context('insert()', () => {
+    it('adds a new node to the linked list when empty', () => {
+      const myLinkedList = new LinkedList()
+      myLinkedList.insert("Azazel")
+      expect(myLinkedList.size()).to.eql(1)
+    })
+    it('adds new node to the end if list is not empty', () => {
```

Comment:
can we separate this out into two different it statements? or not checking the size here if the it statement doesnt include checking for size?